config/application.rb

```diff
@@ -56,6 +56,7 @@ class Application < Rails::Application
     config.cache_store = :memory_store, { size: 64.megabytes }
 
     if ENV['AWS_ACCESS_KEY'].present?
+      Rails.logger.warn "Setting up AWS"
```

Comment:
@jprokay this line is causing the problem with the heroku deploy. please remove it.